### ISSUE TITLE
Documentation needs updating

### ISSUE THREAD

**Issue by mip1983** [danialfarid/ng-file-upload#973]
This is a great library now I've figured things out, but I've faced a lot of time consuming confusion doing things as per the documentation and js-fiddle examples on the home page:

https://github.com/danialfarid/ng-file-upload

Development could do with a pause to bring the docs in line with version 7 (and I guess the existing documentation needs archiving).

**Comment by danialfarid** [comment]
What part is confusing or no in line?

**Comment by mip1983** [comment]
Sorry, part of my confusion was I was using an older version and didn't realize a lot of changes, my fault.

There are only a couple of minor things. 

In one of the release notes for a version it says that ngf-min-size, ngf-max-size are removed (https://github.com/danialfarid/ng-file-upload/releases/tag/6.2.0), but they are still included in main doc and js-fiddle?

There's a mention of 'ngf-accept:'image/*'' on here 'https://github.com/danialfarid/ng-file-upload', which I was trying to use instead of pattern, probably my fault.

There's a mention of "ngf-select ngf-change="upload($file)" on here 'https://github.com/danialfarid/ng-file-upload', which is a little confusing. I was trying ngf-change which didn't work for me, and seems redundant if you can just put the function on ngf-select?

I was just wanting to upload and validate a single image file, but was getting a bit confused whether to use ng-model to a file property on my controller, or ngf-select to pass the file to a method (the later worked for me in the end). I wonder if it's even worth mentioning using the $watch approach, or perhaps moving it to a footnote so there's one prefered clear approach?

**Comment by danialfarid** [comment]
I updated the references to ngf-accept, thanks for pointing it out.
ngf-max-size was removed in version 6.2.0 but was added back in since version 7.0.0. I updated 7.0.0 note to be more clear.
I also removed `ngf-change` from the sample, but it would be useful for the case you have both drop and select on the same element.

I also updated the sample code to be more clear and simpler on how to use it.
